src/tracks/labeltrack/ui/LabelTrackView.cpp

```diff
@@ -1684,7 +1684,7 @@ bool LabelTrackView::DoKeyDown(
       case WXK_NUMPAD_ENTER:
       case WXK_TAB:
          if (mRestoreFocus >= 0) {
-            auto track = *TrackList::Get( project ).Any()
+            auto track = *TrackList::Get(project).Leaders()
```

Comment:
Compensates change of index calculation in `DoAddLabel`

Reply:
Right.  Maybe not obvious.  This change requires the others in handling mRestoreFocus